src/pdngen/test/issue_416.ok

```diff
@@ -0,0 +1,33 @@
+Notice 0: Reading LEF file:  nangate45/NangateOpenCellLibrary.mod.lef
```

Comment:
not convinced "issue_416" is very descriptive; the issues may not have much of a lifetime when we start reorganizing the repos

Reply:
Updated the test to be extend_to_core_ring, which is what the test is checking.